Sample/README.md

```diff
@@ -1,22 +1,29 @@
-# Sample specification v. 0.1
+# Sample Specification
```

Comment:
Are we keeping this generic Sample profile? I thought BioSample was the one to be used in Bioschemas as the generic Sample can be a sample of a new product you get in a supermarket

Reply:
I think we will struggle to get the biosamples team at the EBI to move to the new profile. However, that is not a decision we can make as part of this pull request. It is something for the leadership team and the samples working group to discuss.

For now, I've added links between the two.